src/future/server/shutdown.rs

```diff
@@ -81,28 +81,28 @@ impl Watcher {
 
     fn poll_shutdown_requests(&mut self) -> Poll<Option<()>, ()> {
         Ok(Async::Ready(match try_ready!(self.shutdown_rx.poll()) {
-            Some(tx) => {
-                debug!("Received shutdown request.");
-                self.shutdown = Some(tx);
-                Some(())
-            }
-            None => None,
-        }))
+                            Some(tx) => {
```

Comment:
Hah, oh my god, this can get [really hideous](https://github.com/rust-lang-nursery/rustfmt/issues/1332#issuecomment-289307075).